### PR TITLE
feat: Implement per scene transition

### LINKED ISSUE #271: Implement per scene transition
From Scene 
- get/set Transition (Transition || transitionId)
- get/set TransitionTime (int)

### PR DESCRIPTION
**Pull request by SML-MeSo**
closes #271

### CODE REVIEW

File: src/core/scene.ts [hunk 1]
```diff
@@ -1136,6 +1137,139 @@ export class Scene {
...
+   *
+   * ```javascript
+   * myScene.getTransitionOverride().then(function(transition) {
+   *  // do something here
+   * });
+   * ```
+   */
+  getTransitionOverride(): Promise<Transition> {
+    return new Promise(resolve => {
+      let _sceneId = versionCompare(getVersion()).is.lessThan(sceneUidMinVersion) ? this._id : this._uid;
```
Comment (dcefram) [inline src/core/scene.ts]: I just noticed that almost all other methods had this exact line... couldn't we abstract this into a class property instead?

Comment (SML-MeSo) [reply src/core/scene.ts]: Maybe do it as another issue?
- in order to unclutter changes here
- I think other unmerged changes also use this

Comment (dcefram) [reply src/core/scene.ts]: Yeah, we can do that :)

Comment (SML-MeSo) [reply src/core/scene.ts]: Issue to abstract the above-mentioned line filed as #285. Also, rechecked the branch and unit tests should still be okay. Not really sure why the checks in travis have failed, but I'm confident we can proceed with merging it to 2.10.0 branch

**Pull request merged**